admin/common_auth/views.py

```diff
@@ -0,0 +1,60 @@
+from django.http import HttpResponse
+from django.contrib import messages
```

Comment:
HttpResponse isn't used and so should be removed,
messages is used in the template level - the relevant context will be added in a separate PR (email invitations)